src/EFCore.PG/Diagnostics/NpgsqlEventId.cs

```diff
@@ -17,13 +17,14 @@ namespace Npgsql.EntityFrameworkCore.PostgreSQL.Diagnostics
     /// </summary>
     public static class NpgsqlEventId
     {
-        // Warning: These values must not change between releases.
-        // Only add new values to the end of sections, never in the middle.
-        // Try to use <Noun><Verb> naming and be consistent with existing names.
+        /// <remarks>
+        /// Warning: These values must not change between releases.
+        /// Only add new values to the end of sections, never in the middle.
+        /// Try to use {Noun}{Verb} naming and be consistent with existing names.
+        /// </remarks>
```

Comment:
Not sure that this should be in remarks. @roji?

Reply:
My thinking here is that marking as a `<remark>` makes it visible from intellisense. Plus, since it's on a private type, these shouldn't show up in the auto-generated docs.

Reply:
I agree with @YohDeadfall, xmldocs are for API consumers where as these remarks are for people adding/changing event IDs

Note also that [the same is done in EF Core](https://github.com/aspnet/EntityFrameworkCore/blob/084149d6bd45cc93fc0462fae69dbe03add7e310/src/EFCore.SqlServer/Diagnostics/SqlServerEventId.cs#L22), which we generally try to align to.

Reply:
Makes sense, I'll revert.